huggingface/build_artifacts/inference/config.properties

```diff
@@ -0,0 +1,5 @@
+vmargs=-XX:+UseContainerSupport -XX:InitialRAMPercentage=8.0 -XX:MaxRAMPercentage=10.0 -XX:-UseLargePages -XX:+UseG1GC -XX:+ExitOnOutOfMemoryError
```

Comment:
-XX:-UseContainerSupport? Did we test this configuration?

Reply:
Could we address this comment?

Reply:
Yes, I have done all the current testing with this configuration so far.

Reply:
can you add the suggestion?

Reply:
Replace the option `-XX:+UseContainerSupport` above with `-XX:-UseContainerSupport` .. `+` changes to `-`